### PR TITLE
rand_core 0.6

### PR DESCRIPTION
**Pull request by sergey-shandar**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,15 +11,15 @@ members = [
 [workspace.dependencies]
 blockstack-core = { git = "https://github.com/stacks-network/stacks-blockchain/", branch = "3493-sbtc-peg-out-wire-format" }
 clap = { version = "4.1.1", features = ["derive"] }
-wtfrost = "2.0"
+wtfrost = "3.0.0"
```
Comment (xoloki) [inline Cargo.toml:14]: Did you mean to lock to `3.0.0`?

**Review by xoloki — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (sergey-shandar) [reply Cargo.toml:14]: > Did you mean to lock to `3.0.0`?

no, will fix.

**Review by xoloki — APPROVED** [review]
LGTM

**Review by donpdonp — COMMENTED** [review]
lgtm

**Pull request merged**